events/FlavorTUR.txt

```diff
@@ -5962,7 +5963,7 @@ country_event = {
 		name = flavor_tur.239.a
 		
 		define_heir = {
-			name = "S�leyman Kanuni" 
+			name = "S�leyman Kanuni" 
```

Comment:
what encoding did you use? i told you about the localisation files but forgot about the normal script files. i think they need to be UTF-8 (non BOM) or ANSI
if you used one of those, i'm kinda curious why those lines show as being changed